api/src/main/proto/stellarstation/api/v1/groundstation/groundstation.proto

```diff
@@ -164,6 +164,11 @@ message Plan {
   // Configuration of the radio device used for uplinking to the satellite. Ground stations will
   // need to configure transmission during the plan to match this device.
   radio.RadioDeviceConfiguration uplink_radio_device = 6;
+
+  // Predicted coordinates of the tracked satellite for every second between AOS and LOS. This
```

Comment:
Should we allow for cases where this data isn't needed? In other words, should the request have a parameter to indicate whether it should be returned?

FYI: internally the API used to serve this response will need updated since it doesn't return predictions.